terraform/modules/elasticache/redis.tf

```diff
@@ -0,0 +1,18 @@
+#tfsec:ignore:aws-elasticache-enable-backup-retention:TODO CLDC-2679 setup a snapshot retention limit
+resource "aws_elasticache_cluster" "main" {
+  #checkov:skip=CKV_AWS_134:TODO CLDC-2679 setup a snapshot retention limit
+  #TODO CLDC-2679 setup redis replicas with multi-az and automatic failover
+  #TODO CLDC-2682 setup redis logging
+  cluster_id                 = "${var.prefix}-redis"
+  auto_minor_version_upgrade = true
+  apply_immediately          = true
+  engine                     = "redis"
+  engine_version             = "6.2"

```

Comment:
We use `6.2.6` in PaaS (Terraform here will just take the latest version i.e. `6.2.x`). You can also provide `6.x` which will get the latest minor version, however I'm not sure we'd want to do this in case of compatibility issues